src/js/injection_script.js

```diff
@@ -2711,6 +2711,16 @@ domainBypass("acorta-link.com", () => {
     })
 })
 
+domainBypass("hagamix.blogspot.com", () => {
+    ensureDomLoaded(() => {
+      const gotoLinkButon = document.getElementById("btn-gotolink");
+      setTimeout(function () {
+        document.getElementById("btngetlink").click();
```

Comment:
you want something like: 
`
const btn = document.getElementById('btn-gotolink');
btn.removeAttribute('disabled');
btn.click();
`

Reply:
I know what you mean but it doesn't work. At least not on my Brave Browser. Does it work for you?
You can add my code on "Custom Bypasses" and see for your own. The "click" doesn't get registered strangely and idk why.

Reply:
I checked this in my browser, I haven't checked using ff, but should just work 🤔 I can test tomorrow

Reply:
With in my browser I ment in the console

Reply:
It works in the console but it doesnt work in "custom bypass"

Reply:
this would work regardless, you might want to check this
```suggestion
     domainBypass("hagamix.blogspot.com", () => ensureDomLoaded(() => document.querySelector("button#btn-gotolink").click())
```